misc-tools/judging_statistics

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env php
```

Comment:
This file and `simulate_contest` were accidentally committed.

Reply:
You and thijs found the same issues.